src/wmtk/operations/polygon_mesh/DeleteBubble.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+#include <wmtk/operations/TupleOperation.hpp>
+#include "PolygonMeshOperation.hpp"
+
+namespace wmtk::operations {
+namespace polygon_mesh {
+class DeleteBubble;
+}
+
+template <>
+struct OperationSettings<polygon_mesh::DeleteBubble>
+{
+    // TODO Determine if operation settings are necessary
```

Comment:
ATM you always need to make one somewehre even fi it's empty i think - it's not great :(